core/lexicon/en/policy.inc.php

```diff
@@ -29,7 +30,7 @@
 $_lang['policy_data'] = 'Policy Data';
 $_lang['policy_desc'] = 'Access policies are generic policies that restrict or enable certain actions with MODX.';
 $_lang['policy_desc_name'] = 'The name of the Access Policy';
-$_lang['policy_desc_description'] = 'Optional. A short description of the Access Policy';
+$_lang['policy_desc_description'] = 'Optional. A short description of the Access Policy. Also you might use lexicon keys here.';
```

Comment:
Based on language topics defined in controllers, so I would say that yes, mostly related to access control, but need to check because the manager loads a lot of topics and all of them will be cached.